src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```
@@ -12,6 +12,20 @@
     <RazorComponent Include="**\*.razor" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!--
+      Web assets (js, css, etc under wwwroot) and .razor files are represented as Content items in the Razor SDK, but run foul of
+      checks in the Android and iOS SDK. This annotates these items with metadata that prevents them from being bundled / examined.
+    -->
+    <Content Update="@(Content->HasMetadata('WebContentItem')->WithMetadataValue('WebContentItem', 'true'))"
```

Can you leave off the first item function?
```suggestion
    <Content Update="@(Content->WithMetadataValue('WebContentItem', 'true'))"
```
I think it would still work if that metadata is missing?

---

It seems to work in my limited test. This comment claims otherwise though: https://github.com/dotnet/msbuild/issues/3479#issuecomment-761457919

---

Seems like it should be OK, like it would work for empty metadata:

* https://github.com/dotnet/msbuild/blob/d1f5a8af82872bafadb82227ec99f2fcd7b731ec/src/Build/Evaluation/Expander.cs#L2775-L2779
* https://github.com/dotnet/msbuild/blob/d1f5a8af82872bafadb82227ec99f2fcd7b731ec/src/Utilities/TaskItem.cs#L437

Maybe that comment was using an old version of MSBuild? MAUI will be using MSBuild 17.2 or `dotnet build`, so we should be good.